src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -18,6 +18,9 @@ export type MultiComboboxProps<T> = {
   /** The label associated with this dropdown form element */
   label: string;
 
+  /** Whether the label should get visually hifen */
+  labelHidden?: boolean;
```

Comment:
minor typo here

Reply:
nitpick: can we rename it to `hideLabel`?

Reply:
Done!